wordpress-mobile/WordPress-iOS
# Jetpack Social: Add Jetpack Social no connection view to post settings

See #20784

## Description

- Adds a reusable no social connections view
- Uses the new view on the post settings screen when there are no social connections

## Screenshots

| Light | Dark |
|------|------|
| ![Simulator Screen Shot - Light](https://github.com/wordpress-mobile/WordPress-iOS/assets/2454408/83c170cd-03d2-46cd-b617-1b3fc0800255) | ![Simulator Screen Shot - Dark](https://github.com/wordpress-mobile/WordPress-iOS/assets/2454408/8239ef60-a8bd-4e91-a305-22a67dc554f3) |

## Testing

> **Note**
> At this time, I'm unsure of whether the white border around the icons is intended when in dark mode.

To test:
- Use an account which has no social connections
- Launch Jetpack and login
- Create a blog post or edit one
- On the editor, open the menu (`...`) in the top right
- Tap on `Post Settings`
- **Verify** the no connections view matches (minus the Twitter icon): Th1ahHKq53k5JT1PNDMavY-fi-849_12349
- Switch to a self-hosted site that does not have Jetpack Social
- Repeat the above steps to get to the `Post Settings` screen
- **Verify** the no connections UI does **NOT** appear

## Regression Notes
1. Potential unintended areas of impact
Post settings screen

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Manual testing

3. What automated tests I added (or what prevented me from doing so)
N/A

PR submission checklist:


UI Changes testing checklist:
